### PR TITLE
ci: integrate code-pushup PR comments

### LINKED ISSUE #559: CLI report diff MD formatting (reports-diff.json => reports-diff.md)
Part of epic #478 

### Acceptance criteria

- [x] design and implement `report-diff.md` formatting
- [x] should include categories with changed score
- [x] should include audits with changed score and/or value
- [x] should use table layout for readability

### PR DESCRIPTION
**Pull request by matejchalk**
Closes #559

### CODE REVIEW

**Review by vmasek — APPROVED** [review]

**Comment by vmasek** [comment]
It is even better than I expected 🥳

File: packages/utils/src/lib/reports/__snapshots__/report-diff-mixed.md
Comment (Tlacenka) [inline packages/utils/src/lib/reports/__snapshots__/report-diff-mixed.md:1]: Now that I look at the mixed results, I wonder if we shouldn't sort them in the following way: regressions > improvements > no score change

Currently, regressions and improvements are mixed and without the score column, one may overlook a regression. By having them at the top, all actionable changes are nicely visible.

**Review by Tlacenka — APPROVED** [review]
Looking great!

Love that the comment is updated 😁

File: packages/utils/src/lib/reports/__snapshots__/report-diff-mixed.md
Comment (matejchalk) [reply packages/utils/src/lib/reports/__snapshots__/report-diff-mixed.md:1]: Not sure if that would be better :thinking:  If a major improvement is made, but there's also a minor regression, does it makes sense to list the regression first?

It's ordered by absolute score change (positive or negative), then by value change. The idea is to sort it from highest impact.

In general, regressions can be found by scanning for red badges.

Comment (Tlacenka) [reply packages/utils/src/lib/reports/__snapshots__/report-diff-mixed.md:1]: Let the public decide once it is used by our clients. It is not in scope of this PR anyway.

**Pull request merged**